src/PII/Anonymizer/PasswordAnonymizer.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types=1);
+
+namespace Stefna\Logger\PII\Anonymizer;
+
+final class PasswordAnonymizer implements Anonymizer
+{
+	public const PASSWORD = '_password';
+
+	private const DEFAULT_FIELDS = [
+		self::PASSWORD,
+		'password',
+	];
+
+	/** @var string[] */
+	private $fields;
+
+	public function __construct(string ...$fields)
+	{
+		$this->fields = $fields + self::DEFAULT_FIELDS;
+	}
+
+	public function addField(string $field): void
+	{
+		$this->fields[] = $field;
+	}
+
+	public function support(string $key): bool
+	{
+		return in_array($key, $this->fields, true);
```

Comment:
@sunkan Don't you want to just use `self::PASSWORD` for if key contains this? I doubt we find too many cases where someone adds `_password` as the key.

Reply:
@bix0r check out line `10` and `19`

or am I miss understanding the problem?

Reply:
I don't see how that could catch a field called `user_password`. I guess you should have something like
```
public function support(string $key): bool {
  return in_array($key, $this->fields, true) || strpos($key, self::_PASSWD) !== false;
}
```
Maybe add `user_password` as to the tests?